portfolio.html

```diff
@@ -54,5 +54,127 @@ <h2>I am a Software Developer</h2>
         </div>
       </div>
     </section>
+
+    <section class="works">
+      <div id="set1">
+        <h1>My Recent Works</h1>
+      </div>
+      <div>
+        <hr class="solid" />
+      </div>
+      <div>
+        <img
+          class="set-image"
+          src="/images/ImgPlaceholder.png"
+          alt="works image"
+        />
+      </div>
+      <div id="set2">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <ul>
+          <li><img src="/images/Icons/languages/css.png" alt="csslang" /></li>
+          <li><img src="/images/Icons/languages/html.png" alt="htmllang" /></li>
+          <li>
+            <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+          </li>
+          <li><img src="/images/Icons/languages/ruby.png" alt="rubylang" /></li>
+        </ul>
+      </div>
+      <div class="project-button">
+        <a href="projectlink"
+          ><img
```

Comment:
- [OPTIONAL] I suggest you before merging to add button interactions following the Figma design:

![image](https://user-images.githubusercontent.com/74506933/159082681-f7e15718-bbb5-43ea-8dc9-2c63af29d9b8.png)